Model/View/Asset/Image.php

```diff
@@ -214,9 +214,14 @@ private function convertToReadableFormat($miscParams)
         $miscParams['keep_frame'] = (isset($miscParams['keep_frame']) ? '' : 'no') . 'frame';
         $miscParams['keep_transparency'] = (isset($miscParams['keep_transparency']) ? '' : 'no') . 'transparency';
         $miscParams['constrain_only'] = (isset($miscParams['constrain_only']) ? 'do' : 'not') . 'constrainonly';
-        $miscParams['background'] = isset($miscParams['background'])
-            ? 'rgb' . implode(',', $miscParams['background'])
-            : 'nobackground';
+        if (isset($miscParams['background'])
+            && is_array($miscParams['background'])
+        ){
+            $miscParams['background'] = $miscParams['background']
```

Comment:
What is this change for? Also you seem to still use `$miscParams['background']` for the ternary operator here, which implies an implicit conversion to bool (at least the previous version was clear that it was checking whether the 'background' property was set)

Reply:
only check if the background exists